components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/UserSessionStore.java

```diff
@@ -209,6 +224,7 @@ public String getUserId(String userName, int tenantId, String userDomain) throws
      * @return the list of user Ids of users stored in the given user store
      * @throws UserSessionException if an error occurs when retrieving the user id list from the database
      */
+    @Deprecated
```

Comment:
Better if we can add  why this is deprecated and what should be the alternative method to use (if any)